src/containers/tutor-profile/profile-info/ProfileInfo.styles.js

```diff
@@ -79,5 +79,14 @@ export const styles = {
     description: {
       typography: 'overline'
     }
-  }
+  },
+  wrapperForPhoto: {
+    display: 'flex',
+    gap: '10px'
+  },
+  avatarContainer: {
+    width: '45%'
```

Comment:
it`s not good idea to specify width like this

Reply:
Fixed.